spmd/compiler/distribute.py

```diff
@@ -435,7 +435,7 @@ def remap_arg(arg: object) -> object:
                     # TODO(anj): we need this for getitem but can we be more generic?
                     if isinstance(obj, tuple):
                         return obj
-                    if _get_tracer(obj):
+                    if _get_tracer():
```

Comment:
don't you need obj? why delete it?

Reply:
ah, CommTensor has been updated so this api call no longer uses the obj to determine the slot. 
Rather it now uses get_innermost_proxy_mode() and returns None or mode.tracer. 
(link: https://github.com/pytorch/pytorch/commit/54563e6288efda566be3653a547c8be2781cc547)